flexget/plugins/list/sonarr_list.py

```diff
@@ -208,10 +205,7 @@ def add_show(self, entry):
         show['monitored'] = self.config.get('monitored')
         show['seriesType'] = self.config.get('series_type')
         show['tags'] = self.config.get('tags')
-        # Requires both rootFolder and path otherwise it fails.
-        # takes root_folder_path ID from config and subtracts 1 from it due to how script is setup
-        # since users are passing actual ID it won't be confusing to users
-        show['rootFolderPath'] = rootfolder[self.config.get('root_folder_path') - 1]['path']
+        show['rootFolderPath'] = self.config.get('root_folder_path', rootfolder[0]['path'])
```

Comment:
if the user supplies `root_folder_path` we should probably not query for `rootfolder`, no?

Reply:
explained in gitter.